packages/valory/services/mech/service.yaml

```diff
@@ -253,7 +253,10 @@ type: connection
         poa_chain: ${ETHEREUM_LEDGER_IS_POA_CHAIN:bool:false}
         default_gas_price_strategy: ${ETHEREUM_LEDGER_PRICING:str:eip1559}
       gnosis:
-        address: ${GNOSIS_RPC_0:str:http://host.docker.internal:8545}
+        address: ${GNOSIS_LEDGER_RPC_0:str:http://host.docker.internal:8545}
+        chain_id: ${GNOSIS_LEDGER_CHAIN_ID:int:100}
+        poa_chain: ${GNOSIS_LEDGER_IS_POA_CHAIN:bool:false}
+        default_gas_price_strategy: ${GNOSIS_LEDGER_PRICING:str:eip1559}
```

Comment:
I think these are not necessary, but it's not an issue either:

https://github.com/valory-xyz/mech-predict/blob/d6ec150cf44f52c553c3b0ba48e76c7c27d982c7/packages/valory/agents/mech/aea-config.yaml#L267-L269